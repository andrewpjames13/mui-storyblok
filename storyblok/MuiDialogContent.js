module.exports = {
  "name": "MuiDialogContent",
  "display_name": null,
  "updated_at": "2020-03-31T18:27:06.536Z",
  "schema": {
    "dividers": {
      "type": "boolean"
    },
    "rootClass": {
      "type": "options"
    },
    "content": {
      "type": "bloks",
      "restrict_components": true,
      "component_whitelist": [
        "MuiTypography",
        "MuiExpansionPanel",
        "MuiGrid"
      ]
    }
  },
  "image": null,
  "preview_field": null,
  "is_root": false,
  "preview_tmpl": null,
  "is_nestable": true,
  "all_presets": [],
  "preset_id": null,
  "real_name": "MuiDialogContent",
  "component_group_uuid": null
}