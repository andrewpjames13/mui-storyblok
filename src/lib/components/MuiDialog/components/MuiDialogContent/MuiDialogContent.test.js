import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import MuiDialogContent from './MuiDialogContent';

function setup() {
  const props = {
    content: [{
      component: 'MuiDialogContentTypography',
      content: [{
        component: 'MuiText',
        text: 'text',
      }],
    }],
  };
  const comp = shallow(<MuiDialogContent {...props} />);

  return { comp, props };
}

describe('<MuiDialogContent />', () => {
  it('renders MuiDialogContent', () => {
    const { comp } = setup();
    expect(comp).toBeDefined();
  });

  test('snapshot', () => {
    const { props } = setup();
    const tree = renderer.create(<MuiDialogContent {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
