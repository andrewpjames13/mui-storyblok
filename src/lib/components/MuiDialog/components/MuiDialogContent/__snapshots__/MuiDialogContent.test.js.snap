// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MuiDialogContent /> snapshot 1`] = `
<div
  className="MuiDialogContent-root makeStyles-root-3"
>
  <div
    className="MuiTypography-root makeStyles-root-6 MuiTypography-body1"
  >
    <span
      className="MuiTypography-root makeStyles-root-7 MuiTypography-body1"
    >
       text 
    </span>
  </div>
</div>
`;
